systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -161,4 +164,35 @@ public static boolean isHelmInstall() {
     public static boolean useLatestReleasedBridge() {
         return Environment.BRIDGE_IMAGE.equals(Environment.BRIDGE_IMAGE_DEFAULT);
     }
+
+    private static String getOrDefault(String varName, String defaultValue) {
+        return getOrDefault(varName, String::toString, defaultValue);
+    }
+
+    private static <T> T getOrDefault(String var, Function<String, T> converter, T defaultValue) {
+        String value = System.getenv(var) != null ?
+                System.getenv(var) :
+                (Objects.requireNonNull(JSON_DATA).get(var) != null ?
+                        JSON_DATA.get(var).asText() :
+                        null);
+        T returnValue = defaultValue;
+        if (value != null) {
+            returnValue = converter.apply(value);
+        }
+        VALUES.put(var, String.valueOf(returnValue));
+        return returnValue;
+    }
+
+    private static JsonNode loadJsonEnv() {
```

Comment:
```suggestion
    private static JsonNode loadConfigurationFile() {
```
I would recommend this renaming to make clearer what it is doing, the fact that the file is JSON/YAML is an implementation detail and (hopefully) should be obvious from the documentation + return type